src/main/java/lk/gov/health/phsp/bean/QueryComponentController.java

```diff
@@ -2512,7 +2513,7 @@ public Jpq createClientQuery(QueryComponent qc, Area ccArea, Date ccFrom, Date c
             }
 
             jpql.setJfrom(" from ClientEncounterComponentItem i join i.itemClient c");
-            jpql.setJwhere(" where i.retired=:f ");
+            jpql.setJwhere(" where i.retired=:f and c.reservedClient<>:res ");
```

Comment:
Not sure about param is provided. better to check.

Reply:
As I saw, the same above parameter is used for both places.